src/3d/immediateMode3D.js

```diff
@@ -76,19 +77,19 @@ p5.Renderer3D.prototype.quad = function
 
 p5.Renderer3D.prototype.beginShape = function(mode){
   this.shapeMode = mode;
-  this.verticeStack = [];
+  this.vertexStack = [];
   return this;
 };
 
 p5.Renderer3D.prototype.vertex = function(x, y, z){
-  this.verticeStack.push(x, y, z);
+  this.vertexStack.push(x, y, z);
   return this;
 };
 
 p5.Renderer3D.prototype.endShape = function(){
   var gl = this.GL;
-  this._primitives2D(this.verticeStack);
-  this.verticeStack = [];
+  this._primitives2D(this.vertexStack);
+  this.vertexStack = [];
```

Comment:
super minor, I wonder if we should be creating new arrays here, or just reseting the length to 0 when we want to empty?